scripts/SANS/SANSUtility.py

```
@@ -841,38 +841,32 @@ def get_workspace_type(file_name):
-    @param out_file_name :: the file name of the event data file
-    @param out_file_monitors_name :: the file name of the monitors file
-    @param is_multi_period: if the data set is multiperid
+    @param out_file_path : the file path of the event data file
+    @param out_file_monitors_path : the file name path of the monitors file
+    @param _is_multi_period: if the data set is multiperiod (unused)
     @return the name fo the new group workspace file
     """
+
+    # This is a validation and extracting name out of path
+    full_data_path_name, filename = get_full_path_for_added_event_data(out_file_path)
+    full_monitor_path_name, filename_monitors = get_full_path_for_added_event_data(out_file_monitors_path)
```

Ruff (0.16.1)

[warning] Unpacked variable `filename_monitors` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)